DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/RefreshUIDProcessor.cs

```diff
@@ -0,0 +1,54 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Dicom.Anonymizer.Core.Model;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Processors
+{
+    public class RefreshUIDProcessor : IAnonymizerProcessor
+    {
+        public Dictionary<string, string> ReplacedUIDs { get; } = new Dictionary<string, string>();
+
+        public void Process(DicomDataset dicomDataset, DicomItem item, ProcessContext context = null)
+        {
+            EnsureArg.IsNotNull(dicomDataset, nameof(dicomDataset));
+            EnsureArg.IsNotNull(item, nameof(item));
+
+            string replacedUID;
```

Comment:
>replacedUID [](http://example.com/codeflow?start=19&length=11)

Following else clause doesn't need this variable. Better to move this to if clause or even remove this variable.

Reply:
Thanks, I removed this variable.